packages/mui-base/src/SliderUnstyled/SliderUnstyled.test.tsx

```diff
@@ -10,7 +13,9 @@ import {
   screen,
 } from 'test/utils';
 
-function createTouches(touches) {
+type Touches = Array<{ identifier: number; clientX: number; clientY: number }>;
```

Comment:
Can't we use `TouchEvent` here?

Reply:
@sai6855 Can you resolve this comment? You can instead type it as:
```ts
type Touches = Array<Pick<Touch, 'identifier' | 'clientX' | 'clientY'>>;
```